hosts.go

```diff
@@ -120,6 +120,7 @@ func (h *Hosts) Add(ip string, hosts ...string) error {
 		}
 		h.Lines[position].Hosts = hostsCopy
 		h.Lines[position].Raw = h.Lines[position].ToRaw() // reset raw
+		h.HostsPerLine(9)                                 // make sure this does not break on Windows
```

Comment:
@luthermonson is the actual fix that this line should be added but that it shouldn't be hardcoded to 9?